template/toc.extension.js

```diff
@@ -8,84 +8,98 @@ exports.preTransform = function (model) {
   if (model.items[0].name === 'OpenEphys.Onix1'){
     if (model.items[0].items){
       itemsItemsLength = model.items[0].items.length;
-      let items = [{
-        'name': 'Core Operators',
-        'href' : 'core.html',
-        'topicHref': 'core.html',
-        'topicUid': 'core',
-        'items': []}, {
-        'name': 'Configuration Operators',
-        'href' : 'configure.html',
-        'topicHref': 'configure.html',
-        'topicUid': 'configure',
-        'items': []}, {
-        'name': 'Data I/O Operators',
-        'href' : 'dataio.html',
-        'topicHref': 'dataio.html',
-        'topicUid': 'dataio',
-        'items': []}, {
-        'name': 'Other',
-        'items': [
+      let items = [
+        {
+          'name': 'Core Operators',
+          'href' : 'core.html',
+          'topicHref': 'core.html',
+          'topicUid': 'core',
+          'items': []
+        }, 
+        {
+          'name': 'Configuration Operators',
```

Comment:
This page is for the headstages configuration, but has the same wording as the device configuration page. Specifically, "Device configuration operators" and "configure devices contained".

Wording should be changed so that it is for whatever we have decided the term is for headstages (hubs? aggregates? I forget what the consensus is).

Reply:
The changes I made to try to address this comment are on a different page than the one that is permalinked. I'm going to leave the conversation unresolved if you think my changes adequately address this feedback, feel free to resolve.